.ci_support/linux_64_python3.7.____cpythonsagelib9.7.yaml

```diff
@@ -0,0 +1,33 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '10'
+cdt_name:
+- cos6
+channel_sources:
+- conda-forge
+channel_targets:
+- conda-forge main
+cxx_compiler:
+- gxx
+cxx_compiler_version:
+- '10'
+docker_image:
+- quay.io/condaforge/linux-anvil-cos7-x86_64
+gmp:
+- '6'
+libiconv:
+- '1'
+pin_run_as_build:
+  python:
+    min_pin: x.x
+    max_pin: x.x
+python:
+- 3.7.* *_cpython
+sagelib:
+- '9.7'
```

Comment:
Does this work despite Sage 9.7 dropping Python 3.7 support?

Reply:
That build is being skipped by the `skip: True` rule.

Reply:
I see, checking what (if anything) goes wrong in that combination
might be precisely the purpose of these two files:

- `.ci_support/linux_64_python3.7.____cpythonsagelib9.7.yaml`
- `.ci_support/osx_64_python3.7.____cpythonsagelib9.7.yaml`

Sorry for the noise.

Reply:
No worries. These files should not exist but conda-smithy is not (trying to be) smart enough when `skip: True` has a complicated conditional attached as is the case here.

So, an empty CI run happens that immediately terminates successfully.